src/Components/RelatedCollectionsRail/index.tsx

```diff
@@ -0,0 +1,53 @@
+// import { ArtistCollectionsRailQuery } from "__generated__/ArtistCollectionsRailQuery.graphql"
+import { RelatedCollectionsRailQuery } from "__generated__/RelatedCollectionsRailQuery.graphql"
+import { SystemContextConsumer } from "Artsy"
+import React from "react"
+import { graphql, QueryRenderer } from "react-relay"
+import { RelatedCollectionsRailFragmentContainer as RelatedCollectionsRail } from "./RelatedCollectionsRail"
+
+interface Props {
+  category: string
+  showOnEditorial?: boolean
+}
+
+export const RelatedCollectionsRailContent: React.SFC<Props> = passedProps => {
```

Comment:
`passedProps` -> `props`

Reply:
I noticed I was really only using the `category` prop so I decided to just deconstruct that.

Reply:
👍 If ever wanting to pass the full props object we should still stick with just `props` as naming  since that's the typical react convention. Have noticed `passedProps` often used in tests but that's a different since we're passing props from the `createWrapper` helper function down _to_ the component, which is different from props being referenced inside of the component.